src/test/java/DeckApiHandlerTest.java

```diff
@@ -0,0 +1,67 @@
+
+import org.example.DeckApiHandler;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.net.http.HttpClient;
+
+
+public class DeckApiHandlerTest {
+
+
+    private final HttpClient client = HttpClient.newHttpClient();
+    DeckApiHandler handler = new DeckApiHandler();
+
+    @Test
+    void getDeckHappyPath() {
+        Assertions.assertNotNull(handler.getShuffledDecks(2));
+    }
+
+    @Test
+    void getDecksHappyPath1() {
+        Assertions.assertEquals(200, handler.getShuffledDecks(2).statusCode());
```

Comment:
Is the 200 code here, not equal to check in getDeckHappyPath? You can't get status 200 for a null, so the one above is redundant